scss/_popover.scss

```diff
@@ -1,27 +1,49 @@
+// stylelint-disable custom-property-empty-line-before
+
 .popover {
+  // scss-docs-start popover-css-vars
+  --#{$variable-prefix}popover-zindex: #{$zindex-popover};
+  --#{$variable-prefix}popover-max-width: #{$popover-max-width};
+  @include rfs($popover-font-size, --#{$variable-prefix}popover-font-size);
+  --#{$variable-prefix}popover-bg: #{$popover-bg};
+  --#{$variable-prefix}popover-border-width: #{$popover-border-width};
+  --#{$variable-prefix}popover-border-color: #{$popover-border-color};
+  --#{$variable-prefix}popover-border-radius: #{$popover-border-radius};
+  --#{$variable-prefix}popover-box-shadow: #{$popover-box-shadow};
```

Comment:
This one seems unused, is it needed?

Reply:
Fixed this, missed editing a copy-paste.